docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>"Strings containing <code>://</code> are converted into URLs immediately. Otherwise, they are considered absolute or relative paths and are then interpreted in the context of the client's working directory. " -&gt; "Strings containing <code>://</code> are converted into URLs, while strings without <code>://</code> are considered absolute or relative paths and are thus interpreted in the context of the client's working directory. For example... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>resolved (and this section is reworked in general)</p>